app/main/views/buyers.py

```diff
@@ -16,7 +16,7 @@ def find_buyer_by_brief_id():
     try:
         brief = data_api_client.get_brief(brief_id).get('briefs')
 
-    except:
+    except HTTPError:
```

Comment:
Looks like the broad `try...except` was catching `None` errors on `data_api_client.get_brief(brief_id).get('briefs')` as well. I think `except (HttpError, AttributeError)` would do the trick.

Reply:
Hmm, what would be `None` in that instance? I think get_brief only returns either a valid brief or triggers a 404, so I don't think I can see what would return that. Can you fill me in please? :)

Reply:
Hey yeah weirdly I had a look into this exact line of code yesterday for Kev and Peas PR and the try except has been changed there 😁 Once this is rebased the Try except is a `HTTPError` and this is no longer a problem.
See: https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/331#discussion_r155479224